### PR TITLE
Revoices the resurrected posts using the co-write voice guide

### PR DESCRIPTION
**Pull request by manovotny**
## Summary

First real test of the `/co-write` skill. Clean agents rewrote each resurrected post using only the skill and the voice guide — no session context. Diff against #120 to see what the skill does to 13-year-old writing.

## Changes

- Rewrites `/flash`, `/start`, and `/wp-test` — filler cut, facts and story untouched
- Light pass on `/test-and-troubleshoot-wordpress` — bullets tightened, every `SLIDE_N` placeholder and fact intact
- Rebuilds `/fix-wordpress-admin-styles-not-loading` from `original.md` — the agent's rewrite plus my line edits, now with the actual `wp-config.php` snippet (straight quotes; the live version's curly quotes would fail if pasted)

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: app/(notes)/fix-wordpress-admin-styles-not-loading/page.mdx [hunk 1]
```diff
@@ -25,29 +25,33 @@ After setting up my new computer and getting [MAMP Pro](https://www.mamp.info) r
   width={768}
 />
 
-I tried reinstalling WordPress. No change. I spent a few hours digging through forums, trying countless suggestions. Eventually, something worked.
+Every admin style was gone. Not the evening I had planned.
```
Comment (manovotny) [inline app/(notes)/fix-wordpress-admin-styles-not-loading/page.mdx]: I'd probably cut "Not the evening I had planned." and join the lone sentence with the paragraph below.

File: app/(notes)/fix-wordpress-admin-styles-not-loading/page.mdx [hunk 2]
```diff
@@ -25,29 +25,33 @@ After setting up my new computer and getting [MAMP Pro](https://www.mamp.info) r
...
 
-And change it to:
+Change it to this:
 
 ```php
 error_reporting( E_ALL | E_STRICT );
 ```
 
-After page refresh, the admin styles were back. I even switched the line back to its previous value, and everything still worked. Normally, I discourage editing WordPress core files, but I just needed to get back to work. Fortunately, you don't have to leave the change in place.
+Then refresh the admin screen a few times. The styles come back.
```
Comment (manovotny) [inline app/(notes)/fix-wordpress-admin-styles-not-loading/page.mdx]:
```suggestion
Then refresh the admin screen a few times, and the styles come back.
```

File: app/(notes)/flash/page.mdx [hunk 3]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
 
 <FormattedDate date={data.date} />
 
-My relationship with [Adobe Flash](https://en.wikipedia.org/wiki/Adobe_Flash) started in 2006. I owe a lot to Flash. It has shaped my professional development career for the past six years and paid for our wedding, our house, two cars, and two kids. I have immense gratitude and respect for the platform.
+My relationship with [Adobe Flash](https://en.wikipedia.org/wiki/Adobe_Flash) started in 2006. I owe a lot to Flash. It shaped my career for six years and paid for our wedding, our house, two cars, and two kids. I have immense gratitude and respect for the platform.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: "relationship" sounds funny when talking about a framework.

File: app/(notes)/flash/page.mdx [hunk 4]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
 
-But things change, especially in technology. Today is bittersweet. Today I am officially breaking up with Flash...for good.
+But things change, especially in technology. Today is bittersweet. Today I'm officially breaking up with Flash, for good.
 
-It's not you, Flash. It's me. It is time to see other platforms.
+It's not you, Flash. It's me. It's time to see other platforms.
 
-My wife and I needed money to pay for our wedding, so I started freelancing to raise the extra cash. The HTML, CSS, and JavaScript landscape was completely different six years ago... Does anyone remember slicing an image into nine pieces to get round corners? That kind of frustration made Flash an easy choice at the time. I could simply draw a rounded rectangle on the stage and I was done! Immediate gratification.
+My wife and I needed money for our wedding, so I started freelancing to raise the cash. The HTML, CSS, and JavaScript landscape was a different world six years ago. Remember slicing an image into nine pieces to get rounded corners? That kind of frustration made Flash an easy choice. Draw a rounded rectangle on the stage and you're done. Immediate gratification.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: "My wife and I needed money for our wedding, so I started freelancing to raise the cash."  — completely unnecessary. Cut.

File: app/(notes)/flash/page.mdx [hunk 5]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
+I spent about a year building simple Flash websites, then moved on to advanced Flash applications. Flash was never meant to build websites or applications, but the community took it there anyway. Adobe responded with [Adobe Flex](https://en.wikipedia.org/wiki/Apache_Flex) — still Flash at its core, wrapped in a far richer framework. Flex made Flash a viable enterprise platform, and it took off.
 
-Flex gave me the opportunity to work with some of the best RIA (Rich Internet Application) companies in the world. The experience I gained in learning how to architect complex web apps was invaluable. I am very thankful for the chance to work with some of the best and brightest developers in the industry.
+Flex gave me the chance to work with some of the best RIA (Rich Internet Application) companies in the world. Learning to architect complex web apps was invaluable, and I'm thankful I got to do it alongside some of the best developers in the industry.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: * "RIA (Rich Internet Application)" — By the rules, this should be flipped → "rich internet application (RIA)". Spelled out first, then abbreviated.

File: app/(notes)/flash/page.mdx [hunk 6]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
+The beginning of the end, for me, came in 2010, when Steve Jobs published his [Thoughts on Flash](https://en.wikipedia.org/wiki/Thoughts_on_Flash). That single article is responsible for the current technology landscape, and for the split between Google's perspective and Apple's.
 
-Most of my colleagues fell into the Google camp due to their allegiance to Flash. I took Steve's article as a wake up call. I knew that it would be irresponsible of me to continue to invest my professional career into a language that was dead or dying, so I began to spend any free time I could muster investing in myself and learning other technologies.
+Most of my colleagues fell into the Google camp out of allegiance to Flash. I took Steve's article as a wake-up call. Continuing to invest my career in a dead or dying language would've been irresponsible, so I spent whatever free time I could muster learning other technologies.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: "so I spent whatever free time I could muster learning other technologies." → "so I spent my free time learning other technologies."

File: app/(notes)/flash/page.mdx [hunk 7]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
 
-During the next two years, it would turn out that Steve Jobs was right. In the fall of 2011, [Adobe announced their intention to open source the Flex SDK](https://www.theverge.com/2011/11/18/2570885/adobe-flex-sdk-apache-flash), which I interpreted as a kind way of saying, "We're done with Flash, too". Their [attempt at a FAQ](https://daringfireball.net/linked/2011/11/14/flex) a few days later only confirmed my assumptions. They're done, and shifting focus to HTML5. This only increased my desire and decision to make a transition.
+Over the next two years, Steve Jobs was proven right. In the fall of 2011, [Adobe announced their intention to open source the Flex SDK](https://www.theverge.com/2011/11/18/2570885/adobe-flex-sdk-apache-flash), which I read as a kind way of saying, "We're done with Flash, too". Their [attempt at a FAQ](https://daringfireball.net/linked/2011/11/14/flex) a few days later confirmed it. They're done, and shifting focus to HTML5. My decision to transition got easier.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: * "a kind way of saying" — cut
* "My decision to transition got easier." — "My decision to transition felt validated."

File: app/(notes)/flash/page.mdx [hunk 8]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
 
-These turn of events is what led me to [WordPress](https://wordpress.org) and the more traditional web stack of HTML, CSS, and JavaScript.
+This turn of events led me to [WordPress](https://wordpress.org) and the more traditional web stack of HTML, CSS, and JavaScript.
 
-Technologies like HTML5 and CSS3 have emerged in the wake of this new post-Flash era, with an unrelenting and unapologetic attitude. I have been doing development for 11 years now and I have never experienced such an explosion in development as we are in right now. Yes, I know I am young with even 11 years of experience in the technology landscape, but just look at what is going on now that did not even exist just 2 - 3 years ago. Entire industries and livelihoods have been birthed in such a short time.
+HTML5 and CSS3 have emerged in the wake of this post-Flash era with an unrelenting, unapologetic attitude. In 11 years of development, I've never seen an explosion like the one we're in right now. Eleven years is young in this industry, I know. But look at what exists today that didn't two or three years ago. Entire industries and livelihoods, birthed in a blink:
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: * "with an unrelenting, unapologetic attitude." → "with strong momentum"
* "In 11 years of development, I've never seen an explosion like the one we're in right now. Eleven years is young in this industry, I know." → "In my 11 years of development (I know that's not long), I've never seen an acceleration like the one we're in right now."

File: app/(notes)/flash/page.mdx [hunk 9]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
 
-- The aforementioned [HTML5](https://developer.mozilla.org/en-US/docs/Glossary/HTML5) and [CSS3](https://developer.mozilla.org/en-US/docs/Web/CSS).
+- [HTML5](https://developer.mozilla.org/en-US/docs/Glossary/HTML5) and [CSS3](https://developer.mozilla.org/en-US/docs/Web/CSS).
 - Mobile and tablet platforms, like [iOS](https://www.apple.com/os/ios/) and [Android](https://www.android.com).
 - Responsive frameworks, like [Bootstrap](https://getbootstrap.com) and [Foundation](https://get.foundation).
 - New frameworks, libraries, and plugins released nearly every day.
 
-It really is an exciting time to be a developer and I simply could not continue to try and keep up with where development is going in what little free time I have. The time came to make a dramatic change.
+It's an exciting time to be a developer, and I couldn't keep up with where development is going in what little free time I have. The time came for a dramatic change.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: "It's an exciting time to be a developer, and I couldn't keep up with where development is going in what little free time I have. The time came for a dramatic change." → "It's an exciting time to be a developer, and learning in my free time wasn't enough. It's time for a change."

File: app/(notes)/flash/page.mdx [hunk 10]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
 
-December 31st, 2012 officially marked the last day I will be doing Flash development and I could not be more excited to finally say that.
+December 31st, 2012 officially marked my last day of Flash development. It feels good to finally say that.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: "December 31st, 2012 officially marked my last day of Flash development. It feels good to finally say that." → "December 31st, 2012 marked my last day as a Flash developer, and it feels good to say that."

File: app/(notes)/flash/page.mdx [hunk 11]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
 
-So what's next? I will be turing my focus towards UI / UX front-end development, both in web and mobile. Really looking forwards to taking what I already know about HTML, CSS, JavaScript, and Objective-C to the next level.
+So what's next? I'm turning my focus to UI/UX front-end development, in web and mobile, and building on what I already know about HTML, CSS, JavaScript, and Objective-C.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: "I'm turning my focus to UI/UX front-end development, in web and mobile, and building on what I already know about HTML, CSS, JavaScript, and Objective-C." → I'm turning my focus to UI/UX, frontend development (web and mobile), and building on what I already know — HTML, CSS, JavaScript, and Objective-C.

File: app/(notes)/flash/page.mdx [hunk 12]
```diff
@@ -15,39 +15,39 @@ export const metadata = createMetadata(data);
...
 
-I am really stoked for 2013! It will be a year of immense learning, new opportunities, renewed focus, and incredible growth. I never imagined I would be where I am today as a developer. Flash played an important part of my professional life, and for that, I thank it. But it is time to move on and I cannot wait to see how it all turns out.
+2013 will be a year of learning, new opportunities, and growth. I never imagined I'd be where I am today as a developer. Flash played an important part in that, and I thank it. But it's time to move on, and I can't wait to see how it all turns out.
```
Comment (manovotny) [inline app/(notes)/flash/page.mdx]: * "Flash played an important part in that, and I thank it." → "Flash played an important part in that, and I'm grateful."
* "But it's time to move on, and I can't wait to see how it all turns out." — remove "all"

File: app/(notes)/start/page.mdx [hunk 13]
```diff
@@ -15,20 +15,20 @@ export const metadata = createMetadata(data);
 
 <FormattedDate date={data.date} />
 
-It is no secret that I am _very_ detail oriented, probably to a fault at times. Planning, organizing, and executing is a part of what makes me great at what I do. However, when it comes to launching a website, it is my worst enemy.
+I'm detail oriented, probably to a fault. Planning, organizing, and executing are part of what makes me good at what I do. But when it comes to launching a website, they're my worst enemy.
```
Comment (manovotny) [inline app/(notes)/start/page.mdx]: "But when it comes to launching a website, they're my worst enemy." → "But when it comes to shipping, they can be my worst enemy."

File: app/(notes)/start/page.mdx [hunk 14]
```diff
@@ -15,20 +15,20 @@ export const metadata = createMetadata(data);
...
 
-At the beginning of the year, I decided that sometime in 2012 I wanted to launch my website and begin blogging my thoughts and experiences as a professional developer.
+At the start of the year, I decided 2012 was the year I'd launch my site and start blogging about my experiences as a professional developer.
```
Comment (manovotny) [inline app/(notes)/start/page.mdx]: * "blogging" → "writing"
* "professional" — cut

File: app/(notes)/start/page.mdx [hunk 15]
```diff
@@ -15,20 +15,20 @@ export const metadata = createMetadata(data);
...
 
-The level of entry to launching a website or a blog is incredibly low thanks to many amazing platforms, such as [WordPress](https://wordpress.org). But simply choosing a platform was not what was holding me up. It was me.
+The barrier to launching a blog is low thanks to platforms like [WordPress](https://wordpress.org). But the platform wasn't holding me up. I was.
```
Comment (manovotny) [inline app/(notes)/start/page.mdx]: * "barrier" → "friction"
* "blog" → "site"

File: app/(notes)/start/page.mdx [hunk 16]
```diff
@@ -15,20 +15,20 @@ export const metadata = createMetadata(data);
...
 
-I wanted every aspect of my site to be perfect from day one, so my digital entry to the internet would be as glorious as possible. I wanted every pixel in place, my brand and design completely solidified, and my color palette locked into harmonious perfection. As a developer, I know I selfishly wanted all of those in place to show my capabilities and for visitors to take notice.
+I wanted the site perfect from day one — every pixel in place, the brand solidified, the color palette locked in. My grand entrance to the internet. As a developer, I wanted all of it in place to show what I could do and make visitors take notice.
```
Comment (manovotny) [inline app/(notes)/start/page.mdx:24]: "the color palette locked in" → that's a fantastic edit! Love that you picked "locked in".

File: app/(notes)/start/page.mdx [hunk 17]
```diff
@@ -15,20 +15,20 @@ export const metadata = createMetadata(data);
...
 
-While there is nothing wrong with this, and while I hope to get to that point eventually, all this was doing was holding me up from my goal.
+There's nothing wrong with wanting that, and I hope to get there eventually. But all it was doing was keeping me from the goal.
 
-With year end closing fast, I came to a simple realization... At some point, you just have to _start_.
+With year end closing fast, it hit me: at some point, you have to _start_.
 
-There will never be a "good" time to start. There will always be some unexpected life event or some new idea or opportunity stealing focus. Whether you feel ready or not, you just have to stop making excuses and take that first step. This is me doing just that.
+There's never a good time. There's always an unexpected life event, a new idea, or an opportunity stealing focus. Ready or not, you stop making excuses and take the first step. This is me taking it.
```
Comment (manovotny) [inline app/(notes)/start/page.mdx]: * "There's always an unexpected life event" → "There's always day-to-day life"

File: app/(notes)/start/page.mdx [hunk 18]
```diff
@@ -15,20 +15,20 @@ export const metadata = createMetadata(data);
...
 
-There is nothing fancy here to start... No design. No branding. No carefully selected colors. Just me, my thoughts and ideas, the out-of-the-box minimalistic elegance of Standard, and a handful of carefully selected plugins. The site will take form over time, but for now, I am excited to share what I think and learn as a developer. I hope it will somehow benefit others, start conversations, and be a resource for people.
+There's nothing fancy here. No design. No branding. No carefully selected colors. Me, my thoughts, the out-of-the-box minimalistic elegance of Standard, and a handful of hand-picked plugins. The site will take shape over time. For now, I want to share what I think and learn as a developer, and I hope it helps someone, starts conversations, and becomes a resource.
```
Comment (manovotny) [inline app/(notes)/start/page.mdx]: This isn't on you — you wouldn't know. People wont know that "Standard" is. It was a simple, minimal WordPress theme a startup I was a part of — 8bit — created. I'd rephrase "the out-of-the-box minimalistic elegance of Standard" → "an elegant and  minimalistic WordPress theme"

File: app/(notes)/test-and-troubleshoot-wordpress/page.mdx [hunk 19]
```diff
@@ -16,21 +16,21 @@ export const metadata = createMetadata(data);
 
 <FormattedDate date={data.date} />
 
-I had the incredible honor and privilege to speak at [WordCamp Atlanta](https://2013.atlanta.wordcamp.org/) last week. I gave a presentation on **Testing & Troubleshooting WordPress Plugins & Themes**, which is the culmination of all that I have learned from leading quality at 8BIT for the past three years.
+I spoke at [WordCamp Atlanta](https://2013.atlanta.wordcamp.org/) last week — an honor. My presentation, **Testing & Troubleshooting WordPress Plugins & Themes**, is the culmination of three years leading quality at 8BIT.
```
Comment (manovotny) [inline app/(notes)/test-and-troubleshoot-wordpress/page.mdx]: " — an honor" — feels odd, cut

File: app/(notes)/test-and-troubleshoot-wordpress/page.mdx [hunk 20]
```diff
@@ -40,31 +40,31 @@ SLIDE_5
...
 SLIDE_8
 
 SLIDE_9
 
-- I created [WP Test](/wp-test) to help identify issues while developing WordPress themes and plugins.
-- It is a fantastically exhaustive set of test data to measure the integrity of your plugins and themes.
-- It contains lessons learned from over three years of theme and plugin support, and baffling corner cases, to create a potent cocktail of simulated, quirky user content.
+- I created [WP Test](/wp-test) to catch issues while developing WordPress themes and plugins.
+- It's an exhaustive set of test data to measure the integrity of your plugins and themes.
+- Over three years of theme and plugin support, and its baffling corner cases, distilled into a potent cocktail of quirky, simulated user content.
```
Comment (manovotny) [inline app/(notes)/test-and-troubleshoot-wordpress/page.mdx:60]: Excellent example of a good edit, while keeping some minimal personality. Good job!

File: app/(notes)/test-and-troubleshoot-wordpress/page.mdx [hunk 21]
```diff
@@ -149,45 +149,45 @@ SLIDE_22
 
 SLIDE_23
 
-- At some point in your development career, you will be bit by browser cache.
-- If you are not seeing changes you've made to CSS and JavaScript files, try clearing your browser cache.
+- At some point in your development career, browser cache will bite you.
```
Comment (manovotny) [inline app/(notes)/test-and-troubleshoot-wordpress/page.mdx:152]: Another great edit!

File: app/(notes)/test-and-troubleshoot-wordpress/page.mdx
Comment (manovotny) [inline app/(notes)/test-and-troubleshoot-wordpress/page.mdx:1]: Amazing work on this file in particular! So many great edits.

File: app/(notes)/wp-test/page.mdx [hunk 22]
```diff
@@ -15,48 +15,48 @@ export const metadata = createMetadata(data);
 
 <FormattedDate date={data.date} />
 
-Accounting for all the variables is one of the most challenging aspects of developing for WordPress. There are so many scenarios to account for, such as WordPress versions, plugin versions, plugin conflicts, host server configurations, browsers, and the endless ways you can customize WordPress through post formats, custom post types, theme options, theme templates, etc.
+Building for WordPress means accounting for endless variables: WordPress versions, plugin versions, plugin conflicts, host configurations, browsers, and all the ways people customize WordPress — post formats, custom post types, theme options, theme templates.
 
-In this vast sea of variety is both the pleasure and the pain in developing WordPress themes and plugins for mass distribution.
+That variety is both the pleasure and the pain of shipping themes and plugins for mass distribution.
```
Comment (manovotny) [inline app/(notes)/wp-test/page.mdx]: "pleasure" → "fun"

File: app/(notes)/wp-test/page.mdx [hunk 23]
```diff
@@ -15,48 +15,48 @@ export const metadata = createMetadata(data);
...
 
-Today I would like to begin to put an end to one of those variables. I would like to introduce the release of [WP Test](https://wptest.io), the best tests for [WordPress](https://wordpress.org).
+Today I want to start putting an end to one of those variables. Meet [WP Test](https://wptest.io), the best tests for [WordPress](https://wordpress.org).
```
Comment (manovotny) [inline app/(notes)/wp-test/page.mdx:22]: "Meet" is such a great edit!

File: app/(notes)/wp-test/page.mdx [hunk 24]
```diff
@@ -15,48 +15,48 @@ export const metadata = createMetadata(data);
...
 
-I purposely excluded the word "comprehensive" in the description of the tests. I know the test data is most likely missing coverage in some areas. That's where the community comes in. Let me know of a test that is not covered and I will get it added to the data so we can all account for it in our products.
+I left the word "comprehensive" out of the description on purpose. The test data is most likely missing coverage somewhere — that's where the community comes in. Tell me about a test that isn't covered and I'll add it to the data so we can all account for it in our products.
 
 ## Life
 
-Testing can be tedious and boring enough, but there is no need for the test data itself to be mundane. Lorem ipsum is effective. It does simulate text flow fairly well, but it's boring as hell.
+Testing is tedious enough without mundane test data. Lorem ipsum works (it simulates text flow well), but it's boring as hell.
```
Comment (manovotny) [inline app/(notes)/wp-test/page.mdx]: "as hell" — I can't believe I wrote this! I would never swear in a blog post now — ever! Cut.

File: app/(notes)/wp-test/page.mdx [hunk 25]
```diff
@@ -15,48 +15,48 @@ export const metadata = createMetadata(data);
...
 
-## The Future
+## The future
 
-WP Test would be good enough as just test data, but I do have bigger plans for what it could be beyond what it is today. I would love to see it grow into an all encompassing suite of WordPress testing tools and data. Who knows if it will ever get there, but at the very least, now we can all share and contribute to an impressive set of test data.
+WP Test would be good enough as test data alone, but I have bigger plans for it. I'd love to see it grow into an all-encompassing suite of WordPress testing tools and data. Who knows if it will get there — at the very least, we can all share and contribute to an impressive set of test data now.
 
-Please let me know if you have any ideas, additional test cases, or feedback! Let's make WordPress testing easier and resilient together.
+Have ideas, additional test cases, or feedback? Let me know. Let's make WordPress testing easier and more resilient together.
```
Comment (manovotny) [inline app/(notes)/wp-test/page.mdx]:
```suggestion
Have ideas, additional test cases, or feedback? [Let me know](https://x.com/intent/post?screen_name=manovotny). Let's make WordPress testing easier and more resilient together.
```

File: app/(notes)/wp-test/page.mdx
Comment (manovotny) [inline app/(notes)/wp-test/page.mdx:1]: This was another fantastic edit!
